WMFComponents/Sources/WMFComponents/Components/Activity/WMFActivityView.swift

```diff
@@ -0,0 +1,154 @@
+import SwiftUI
+import WMFData
+
+public struct WMFActivityView: View {
+    @ObservedObject var appEnvironment = WMFAppEnvironment.current
+
+    var theme: WMFTheme {
+        return appEnvironment.theme
+    }
+
+    @ObservedObject var viewModel: WMFActivityViewModel
+
+    public init(viewModel: WMFActivityViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    private var titleStyles: HtmlUtils.Styles {
+        HtmlUtils.Styles(font: WMFFont.for(.title3), boldFont: WMFFont.for(.boldTitle3), italicsFont: WMFFont.for(.italicGeorgiaTitle3), boldItalicsFont: WMFFont.for(.boldItalicGeorgiaTitle3), color: theme.text, linkColor: theme.link, lineSpacing: 3)
+    }
+    
+    public var body: some View {
+
+        if viewModel.isLoggedIn {
+            VStack(alignment: .leading) {
+                WMFHtmlText(html: viewModel.localizedStrings.getGreeting(), styles: titleStyles)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .padding(.top, 24)
+                    .padding(.bottom, 8)
+                
+                if let editActivityItem = viewModel.editActivityItem {
+                    WMFActivityComponentView(
+                        activityItem: editActivityItem,
+                        title: viewModel.title(for: editActivityItem.type),
+                        onButtonTap: viewModel.action(for: editActivityItem.type),
+                        shouldDisplayButton: true,
```

Comment:
I notice all of these `shouldDisplayButton` are true. Is there an instance where it needs to be false? If not maybe we can remove that property.

Reply:
It was for an earlier version of the prototype where the button could be hidden, I'll remove it.